composer.json

```diff
@@ -115,6 +115,7 @@
         "symfony/psr-http-message-bridge": "^7.0"
     },
     "conflict": {
+        "illuminate/collections": "<11.0.0",
```

Comment:
`laravel/prompts` deps with `minimum-stability: dev` and `prefer-stable: true` will resolved `illuminate/collections` 10.

Reply:
![CleanShot 2024-01-22 at 13 04 18](https://github.com/laravel/framework/assets/172966/118d2a1a-43a6-401d-b323-a031caa5f180)